src/rekapi.actor.js

```diff
@@ -659,9 +587,9 @@ rekapiModules.push(function (context) {
    */
   Actor.prototype.getKeyframeProperty = function (property, millisecond) {
     var propertyTrack = this._propertyTracks[property];
-    if (propertyTrack) {
-      return _.findWhere(propertyTrack, { millisecond: millisecond });
-    }
+    var index = propertyIndexInTrack(propertyTrack, millisecond);
+    if (index != null)
```

Comment:
I use val != null to check for both null and undefined.  Probably just want to check for undefined in this instance.